portugues/colabora/comdiferencas/diff/pull/reenvio/watch/emprego/emprego.md

```diff
@@ -1,4 +1,13 @@
 Você foi aprovado em uma entrevista de emprego na empresa cujo nome possui três letras
  pelo fato de ter contribuido para a comunidade open source, através de correções de bugs 
-e criação de novas features em aplicações de terceiros:
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.